code/go/0chain.net/blobbercore/handler/handler.go

```diff
@@ -239,6 +239,20 @@ func ObjectTreeHandler(ctx context.Context, r *http.Request) (interface{}, error
 	return response, nil
 }
 
+func RefsHandler(ctx context.Context, r *http.Request) (interface{}, error) {
+	if r.Method == "POST" {
+		return nil, common.NewError("invalid method", "Invalid method used. Use GET instead")
```

Comment:
Have this check in the router itself, gorilla is capable of specifyng request type with the handlers

Reply:
Done